docs/philosophy.mdx

```diff
@@ -0,0 +1,11 @@
+---
+id: philosophy
+title: Philosophy
+slug: /philosophy
+---
+
+Software should be a tool that allows others to reach their goals more efficiently and effectively. It should be thoughtfully designed, accessible, unobtrusive, and opinionated for the purpose it serves. We work with the Product Team and Stakeholders to accomplish out results.
```

Comment:
Update "out" to "our" in the following sentence: "We work with the Product Team and Stakeholders to accomplish out results."